e2e/metrics/bundledeployment_test.go

```diff
@@ -24,29 +23,38 @@ var _ = Describe("BundleDeployment Metrics", Label("bundledeployment"), func() {
 		// objName is going to be "randomized" instead of using a dedicated and
 		// random namespace, like it is the case for the other tests.
 		objName string
+
+		r = rand.New(rand.NewSource(GinkgoRandomSeed()))
 	)
 
 	BeforeEach(func() {
 		k = env.Kubectl.Namespace(env.Namespace)
 		kw = k.Namespace(namespace)
```

Comment:
I don't understand how this code could have passed the tests. `kw` does not have `targetNS` as namespace.